Description: lintian reports a typo in libXm. This patch fixes it.
Author: Paul Gevers <elbrus@debian.org>
Forwarded: http://bugs.motifzone.net/show_bug.cgi?id=1589
Last-reviewed: 06 Feb 2013

--- a/lib/Xm/FontS.c
+++ b/lib/Xm/FontS.c
@@ -1225,7 +1225,7 @@
 
     if ((fontdata = XLoadQueryFont(XtDisplay((Widget) fsw), font)) == NULL)
     {
-	sprintf(buf, "Font '%s'\nis not availiable on this machine", font);
+	sprintf(buf, "Font '%s'\nis not available on this machine", font);
 	DisplayUserError(fsw, buf);
 	err = True;
     }
@@ -1275,7 +1275,7 @@
 					   fbuf)) == NULL) 
 	    {
 		sprintf(buf,
-			    "Font '%s'\nis not availiable on this machine",
+			    "Font '%s'\nis not available on this machine",
 			    font);
 		DisplayUserError(fsw, buf);
 		    err = True;
